src/status_im2/navigation/events.cljs

```diff
@@ -21,7 +22,9 @@
   (rf/merge
    cofx
    {:db         (all-screens-params db go-to-view-id screen-params)
-    :dispatch-n [[:hide-bottom-sheet]]}
+    :dispatch-n [[:hide-bottom-sheet]
+                 (when (= go-to-view-id shell.constants/community-screen)
```

Comment:
I see this `navigate-to` event as a general utility, so we shouldn't couple it with specific screens.

We can probably dispatch this event when the community overview screen is loaded, more or less here https://github.com/status-im/status-mobile/blob/aa6d44bee18c92a39c6e6da8b2199cb6dadd5647/src/status_im2/contexts/communities/overview/view.cljs#L359

You will need to convert the component to a Reagent form-2 component. It's a common technique we use to do something effectful when some component is mounted.

```clojure
(defn overview
  []
  (rf/dispatch :mark-as-read)
  (fn [id]
    ...))
```